decidim-elections/spec/models/decidim/elections/question_spec.rb

```diff
@@ -17,6 +17,25 @@ module Elections
         expect(subject.election).to eq(election)
       end
 
+      it "has many response options" do
+        expect(subject.response_options.count).to be_positive
+        expect(subject.response_options_count).to be_positive
```

Comment:
Maybe we could add an additional expectation like?
 
```suggestion
        expect(subject.response_options_count).to be_positive
        expect(subject.response_options_count).to eq(subject.response_options.count)
```

Reply:
yep